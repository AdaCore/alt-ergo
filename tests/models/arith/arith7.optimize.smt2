(set-logic ALL)
(set-option :produce-models true)
(declare-const x Real)
(push 1)
(assert (< x 0.5))
(maximize x)
(check-sat)
(get-model)
(pop 1)
(push 1)
(assert (< x (- 0.0 0.5)))
(maximize x)
(check-sat)
(get-model)
(pop 1)
