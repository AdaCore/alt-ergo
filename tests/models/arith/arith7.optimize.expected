
unknown
(
  (define-fun x () Real 0)
)

unknown
(
  (define-fun x () Real (- (/ 3 2)))
)
